services/email.js

```diff
@@ -9,9 +9,11 @@ export default {
     name: 'email',
     // At least one of the CRUD methods is Required
     create: (req, resource, params, body, config, callback) => {
-        if (!params.from || !params.to || !params.subject || !params.message)
+        if (!params.subject || !params.message)
             return callback('Error: Missing parameters', null);
 
+        const to = SMTP.to, from = SMTP.from;
```

Comment:
I see now about the from thing, OK, although I am not sure about the spam thing you mention, but I agree let's play it safe for now :)